mllib/src/main/scala/org/apache/spark/ml/feature/StringIndexer.scala

```diff
@@ -65,12 +66,15 @@ class StringIndexer(override val uid: String) extends Estimator[StringIndexerMod
   def this() = this(Identifiable.randomUID("strIdx"))
 
   /** @group setParam */
+  def setSkipInvalid(value: Boolean): this.type = set(skipInvalid, value)
```

Comment:
Do you have a specific example in mind?

My suggestion was because `SkipInvalid` has nothing to do with the fitting routine itself; rather, it affects how the model is evaluated.